### PR TITLE
BAM-1606: Publisher events API documentation

### PR DESCRIPTION
**Pull request by csdev**

### CODE REVIEW

File: docs/tagpublishercustom.rst [hunk 1]
```diff
@@ -171,8 +223,8 @@ response::
 
     https://ad.doubleclick.net.example/ddm/trackimp/N1234.1234567NARRATIV/B12345678.123456789;dc_trk_aid=123456789;dc_trk_cid=12345678;ord={RAND};dc_lat=;dc_rdid=;tag_for_child_directed_treatment=?"
 
-To fire an event tracker, insert a new hidden HTML ``IMG`` tag with the
-tracker URL as the image source::
+To fire a third-party event tracker, insert a new hidden HTML ``IMG`` tag
+with the tracker URL as the image source::
 
     <IMG SRC="https://ad.doubleclick.net.example/ddm/trackimp/N1234.1234567NARRATIV/B12345678.123456789;dc_trk_aid=123456789;dc_trk_cid=12345678;ord=1629147433127336253;dc_lat=;dc_rdid=;tag_for_child_directed_treatment=?" />
```
Comment (myfashionhub) [inline docs/tagpublishercustom.rst]: `img` and `src` are lowercase

**Comment by pbecotte** [comment]
If implemented correctly they should create a new UUID on each navigation before firing a new page impression. Does that seem clear to you from the description of what a UUID is?

**Comment by lat2118** [comment]
​Yes!​


Warm Regards,
Laura Taylor

*Marketing & HR Manager*
335 Madison Avenue, 16F | New York, NY, 10017
(562) 243 - 8474  |
​Narrativ.com
<https://mailtrack.io/trace/link/7f98b0a46b773d270655cd67e7ea4f33449f1be4?url=http%3A%2F%2Fnarrativ.com%2F&userId=1980714&signature=366f85cbce84c394>

*​​2017 Digiday Signal Award Winner:*
<https://mailtrack.io/trace/link/6a4323947f5998685c4e7156fe2d599eee122cdd?url=https%3A%2F%2Fstatic1.squarespace.com%2Fstatic%2F55db494de4b0f408c8e00f07%2Ft%2F59839176f7e0ab6f61d3385f%2F1501794681117%2FDigiday%2BSignal%2BAwards%2B2017%2B%253A%2BDermstore%2BCase%2BStudy.pdf&userId=1980714&signature=85029d2a06a0c233>
Best Native/Content Advertising Platform of the Year

File: docs/tagpublishercustom.rst (hunk 1 above)
Comment (csdev) [reply docs/tagpublishercustom.rst]: will fix in a subsequent PR

**Pull request merged**